public/static/detach.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
```

Comment:
we've gotta just make this a one liner to copy/paste. no one wants to run a random script downloaded from the internet